<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clasificado</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="forlogo.qrc">:/newPrefix/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Classificado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>162</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A Project By University Of Lahore</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Islamabad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pakistan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prepros">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pre-Processing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistical Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visu">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="classify">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Classification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertion">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>.ARFF to .CSV Converssion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="forlogo.qrc"/>
 </resources>
 <connections/>
</ui>
